app/src/androidTest/java/com/duckduckgo/app/statistics/VariantManagerTest.kt

```diff
@@ -43,96 +43,6 @@ class VariantManagerTest {
         assertEquals(0, variant.features.size)
     }
 
-    // Notification Drip Experiment
-
-    @Test
-    fun notificationDripTestControlGroupVariantIsNotActive() {
```

Comment:
I guess it depends. Generally I think we have two phases:

1. Remove the experiment: just change the weight and amend the variant manager tests.
2. Delete the experiment: delete variants, tests around variants, etc.

This is more like a phase 2 because we already changed the weights to 0.0 a while back.